.gitmodules

```diff
@@ -88,3 +88,6 @@
 [submodule "kits/clevertap-kit"]
 	path = kits/clevertap-kit
 	url = git@github.com:mparticle-integrations/mparticle-android-integration-clevertap.git
+[submodule "kits/pilgrimsdk-kit"]
+	path = kits/pilgrimsdk-kit
+	url = https://github.com/foursquare/mparticle-android-integration-example
```

Comment:
Looks like the rest of these are git links, not http. should probably match

Reply:
Will do

Reply:
also seems to point to the `mparticle-integrations:mparticle-android-integration-foursquare`